cla_frontend/templates/base.html

```diff
@@ -2,7 +2,8 @@
 <!DOCTYPE html>
 <html lang="en" {% block html_attrs %}ng-csp{% endblock %}
   data-analytics-id="{{ analytics_id }}"
-  data-analytics-domain="{{ analytics_domain }}">
+  data-analytics-domain="{{ analytics_domain }}"
+  cla-features="{{ cla_features|join:' ' }}">
```

Comment:
My hunch is that this would be better with the `data-` prefix but I don't know if there's any good reasoning to that hunch tbh

Reply:
That's a good suggestion. Updated attribute with data- prefix